src/core/client/admin/routes/Moderate/Queue/ModerationQueue.tsx

```diff
@@ -0,0 +1,32 @@
+import { useRouter } from "found";
+import { FunctionComponent, useEffect } from "react";
+
+import { getModerationLink } from "coral-framework/helpers";
+
+interface Props {
+  mode: "PRE" | "POST" | "%future added value" | null;
+  siteID?: string;
+  storyID?: string;
+}
+
+const ModerationQueue: FunctionComponent<Props> = ({
+  mode,
+  siteID,
+  storyID,
+}) => {
+  const { router } = useRouter();
+  useEffect(() => {
+    if (mode) {
+      router.replace(
+        getModerationLink({
+          queue: mode === "PRE" ? "pending" : "reported",
+          storyID,
+          siteID,
+        })
```

Comment:
You should also handle the `section` as well if you can. See `parseModerationOptions` to see how to extract that from the `match` parameter.